### PR TITLE
PLAT-136780: Update LabeledIconButton

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
LabeledIconButton component needs to be updated to the latest design changes

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added styling for LabeledIconButton based on the latest UX design for Silicon

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Some screenshot tests fail for other skins, but now the label position and button displays correctly for them.

### Links
[//]: # (Related issues, references)
PLAT-136780

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

File: LabeledIconButton/LabeledIconButton.js [hunk 1]
```diff
@@ -203,6 +203,7 @@ const LabeledIconButtonBase = kind({
 					pressed={pressed}
 					selected={selected}
 					spriteCount={spriteCount}
+					css={css}
```
Comment (stanca-pop-lgp) [inline LabeledIconButton/LabeledIconButton.js:206]: Please sort alphabetically.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: LabeledIconButton/LabeledIconButton.module.less [hunk 2]
```diff
@@ -23,28 +23,271 @@
...
+		}
+
+		&.smallest,
+		&.small,
+		&.large,
+		&.huge {
+			&.left,
+			&.right,
+			&.before,
+			&.after {
```
Comment (MikyungKim) [inline LabeledIconButton/LabeledIconButton.module.less:75]: Is this needed for increasing specificity?

**Review by MikyungKim — APPROVED** [review]
I think we need to remove the margin below.
image

Except for that, LGTM.

**Comment by daniel-stoian-lgp** [comment]
> I think we need to remove the margin below.
> image
> 
> Except for that, LGTM.
> Please resolve the merge conflict.

You are right!
It seems this margin comes from LabeledIcon, and according the UI guide, it should be 0. LabeledIcon must be fixed too. 
I can modify it.

**Review by jeonghee27 — APPROVED** [review]
LGTM
I hope someday it can be cleaned more simply.

**Pull request merged**